<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferencesWindow</class>
 <widget class="QMainWindow" name="PreferencesWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QCheckBox" name="useNativeToolbarCheckBox">
      <property name="text">
       <string>Use native NSToolbar</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <widget class="QWidget" name="generalPage">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="generalLabel">
          <property name="text">
           <string>General preferences would go here</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="networkPage">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="networkLabel">
          <property name="text">
           <string>Network preferences would go here</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="advancedPage">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="advancedLabel">
          <property name="text">
           <string>Advanced preferences would go here</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionGeneral"/>
   <addaction name="actionNetwork"/>
   <addaction name="actionAdvanced"/>
  </widget>
  <action name="actionGeneral">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="macunifiedtoolbar.qrc">
     <normaloff>:/qtlogo.png</normaloff>:/qtlogo.png</iconset>
   </property>
   <property name="text">
    <string>General</string>
   </property>
  </action>
  <action name="actionNetwork">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="macunifiedtoolbar.qrc">
     <normaloff>:/qtlogo.png</normaloff>:/qtlogo.png</iconset>
   </property>
   <property name="text">
    <string>Network</string>
   </property>
  </action>
  <action name="actionAdvanced">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="macunifiedtoolbar.qrc">
     <normaloff>:/qtlogo.png</normaloff>:/qtlogo.png</iconset>
   </property>
   <property name="text">
    <string>Advanced</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="macunifiedtoolbar.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionGeneral</sender>
   <signal>triggered()</signal>
   <receiver>PreferencesWindow</receiver>
   <slot>toolbarItemTriggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionNetwork</sender>
   <signal>triggered()</signal>
   <receiver>PreferencesWindow</receiver>
   <slot>toolbarItemTriggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAdvanced</sender>
   <signal>triggered()</signal>
   <receiver>PreferencesWindow</receiver>
   <slot>toolbarItemTriggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useNativeToolbarCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>PreferencesWindow</receiver>
   <slot>useNativeToolBarToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>61</x>
     <y>87</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>toolbarItemTriggered()</slot>
  <slot>useNativeToolBarToggled(bool)</slot>
 </slots>
</ui>
